README.md

```diff
@@ -185,6 +185,7 @@ The following table lists the global, default, and other parameters supported by
 | `global.url` | Public URL of the Mender Server, replace with your domain | `https://mender-api-gateway` |
 | `default.affinity` | Optional affinity values that applies to all the resources | `nil` |
 | `default.toleration` | Optional toleration values that applies to all the resources | `nil` |
+| `default.hpa` | HorizontalPodAutoscaler support | `nil` |
```

Comment:
The documentation for the options under `hpa` are not documented here.

Reply:
Right! Added